packages/scaffolder-cli/test-results/.last-run.json

```diff
@@ -0,0 +1,4 @@
+{
+  "status": "passed",
+  "failedTests": []
+}
```

Comment:
Should this file be removed (and excluded in .gitignore)? Should the results be written somewhere else in the first place, not in the package itself? Some common standard location for state storage - possibly like `~/.scaffolder-cli-state` or whatever